src/Common/src/System/Globalization/IdnMapping.cs

```diff
@@ -117,7 +117,7 @@ public string GetUnicode(string ascii, int index, int count)
             if (index < 0 || count < 0)
                 throw new ArgumentOutOfRangeException((index < 0) ? nameof(index) : nameof(count), SR.ArgumentOutOfRange_NeedNonNegNum);
             if (index > ascii.Length)
-                throw new ArgumentOutOfRangeException("byteIndex", SR.ArgumentOutOfRange_Index);
+                throw new ArgumentOutOfRangeException(nameof(byteIndex), SR.ArgumentOutOfRange_Index);
```

Comment:
@stephentoub Sorry, I'm still running the build on my machine... I know it shouldn't be an excuse, but the repo takes so long to compile :cry: